package.json

```diff
@@ -38,13 +38,11 @@
   },
   "dependencies": {
     "chalk": "^1.0.0",
+    "generator-karma": ">=0.9.0",
```

Comment:
should not be added to deps: https://github.com/yeoman/generator-angular#usage

`generator-karma` is a global pack